pt-BR/working_with_javascript_in_rails.md

```diff
@@ -1,62 +1,62 @@
 **NÃO LEIA ESTE ARQUIVO NO GITHUB, OS GUIAS SÃO PUBLICADOS NO https://guiarails.com.br.**
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
-Working with JavaScript in Rails
+Trabalhando com JavaScript no Rails
 ================================
 
-This guide covers the built-in Ajax/JavaScript functionality of Rails (and
-more); it will enable you to create rich and dynamic Ajax applications with
-ease!
+Este guia aborda as funcionalidades internas Ajax/javascript do Rails (e
+mais); Isso permitirá que você crie aplicações Ajax ricas e dinâmicas com
+facilidade!
 
-After reading this guide, you will know:
+Apoós ler este guia, você saberá:
 
-* The basics of Ajax.
-* Unobtrusive JavaScript.
-* How Rails' built-in helpers assist you.
-* How to handle Ajax on the server side.
-* The Turbolinks gem.
-* How to include your Cross-Site Request Forgery token in request headers
+* O basico de *Ajax*. 
+* *Unobtrusive* JavaScript.
```

Comment:
Unobtrusive é uma palavra pra gente pensar a respeito @campuscode/guia-rails-team ... Talvez 'não obtrusivo' seja melhor do que deixar o termo em inglês.

Reply:
o que você acha da gente usar: **JavaScript discreto** ?

Reply:
https://es.wikipedia.org/wiki/JavaScript_no_obstructivo

https://docs.microsoft.com/pt-br/aspnet/mvc/overview/older-versions/creating-a-mvc-3-application-with-razor-and-unobtrusive-javascript

Eu não tenho muita opinião sobre o tema mas tentei encontrar algumas referências desse termo mas foi bem difícil achar algo traduzido a maioria das pessoas parece manter o **unobtrusive**. 

A gente pode fazer um **termo português (termo inglês)** que é algo que temos feito em outros termos mais complexos.

Reply:
então deixa como está?

Reply:
Ficaria algo como `JavaScript não obstrusivo (*unobtrusive*)`

Reply:
obrigado! feito!